packages/fluent/scss/toolbar/index.scss

```diff
@@ -0,0 +1,29 @@
+// Module meta
+$_kendo-module-meta: (
+    name: "toolbar",
+    dependencies: (
+        "button",
+        "popup",
+        "menu",
+        "icon",
+        "list"
```

Comment:
I think list dependency is not needed here. The overflow popup utilizes the menu-item styles.